__tests__/app.test.js

```diff
@@ -78,3 +78,38 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("GET /api/articles", () => {
+  test("200: returns all the articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(Array.isArray(articles)).toBe(true);
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face: